app/components/Header/index.js

```diff
@@ -81,12 +89,43 @@ function AccountDropdownItems({
 class Header extends Component<Props, State> {
   state: State = {
     accountOpen: false,
-    shake: false
+    shake: false,
+    forgotPassword: false,
```

Comment:
This should probably be something like an enum instead of multiple booleans, since only one of them can be active at a time. You could do something like: `mode: 'login' | 'register' | 'forgotPassword'`, and flow will make sure that those are the only valid values.

Reply:
Something like this?
<code>
type State = {
  accountOpen: boolean,
  shake: boolean,
  mode: 'login' | 'register' | 'forgotPassword'
};
</code>
and then
<code>
  state: State = {
    accountOpen: false,
    shake: false,
    mode: 'login'
  };
</code>
then use a switch case when deciding what form will be displayed?

Reply:
yep!